src/video-player.js

```diff
@@ -612,6 +624,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
 
   playlist(sources, options = {}) {
     this._initQualitySelector();
+    this._sendInternalAnalytics();
```

Comment:
should not we send any param here?

Reply:
we get there options that are not listed in analytics params for init action